src/rust/src/x509/sct.rs

```diff
@@ -49,11 +49,105 @@ pub(crate) enum LogEntryType {
     PreCertificate,
 }
 
+// This trait exists solely to lift the bodies of `hash_algorithm`
+// and `signature_algoritm` out of a PyO3 context, for code coverage.
+pub(crate) trait ToAttr {
```

Comment:
Why does this need to be a trait, can't these just be functions on their respective types?

Reply:
My contrived reason is that it's a behavior description, but I don't have a really strong justification. I could make them into associated functions instead.

Reply:
There's no places we're generic over the trait, so to me it doesn't seem to add much value.

Reply:
Sounds good, will remove now.